#! /bin/sh /usr/share/dpatch/dpatch-run
## gtkhtml-sharp-3.14_port.dpatch by Tim Retout <tim@retout.co.uk>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Port to gtkhtml-sharp-3.14

@DPATCH@
diff -urNad f-spot~/src/InfoDisplay.cs f-spot/src/InfoDisplay.cs
--- f-spot~/src/InfoDisplay.cs	2008-03-09 11:12:45.000000000 +0000
+++ f-spot/src/InfoDisplay.cs	2008-03-20 19:31:22.000000000 +0000
@@ -148,7 +148,7 @@
 
 		private void Update ()
 		{
-			Gtk.HTMLStream stream = this.Begin (null, "text/html; charset=utf-8", Gtk.HTMLBeginFlags.Scroll);
+			Gtk.HTMLStream stream = this.Begin (null, "text/html; charset=utf-8", Gtk.HTMLBeginFlags.KeepScroll);
 			
 			string bg = Color (this.Style.Background (Gtk.StateType.Active));
 			string fg = Color (this.Style.Foreground (Gtk.StateType.Active));
